SU-HSDO/suhumsci
# Update drupal core 8.8

# READY FOR REVIEW

# Summary
- Update core and all modules to the latest version.
- Eliminate `config_installer` profile and go with native drush install from config
- moved default content into a hidden module that shouldn't be installed at any point. Its only used to manually import the default content.

# Need Review By (Date)
- 01/14

# Urgency
- Medium

# Steps to Test
1. Checkout this branch
1. `composer install --prefer-source`
1. `drush si su_humsci_profile -y`
1. verify site install was successful
1. verify home page has simple default content.
1. make a basic page to verify nothing is broken.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
